### DISCUSSION TITLE
Flux assets in virtual directory

### DISCUSSION THREAD

**Discussion by bebo925** [livewire/flux#818] [Feature Requests]
I am restricted to serving my application under a virtual directory ie. https://my-domain.com/fluxed-application.  I can't find a way to configure the flux assets to not point to the root url.  

I'm getting https://my-domain.com/fluxed-application/flux/flux.min.js.
I need https://my-domain.com/fluxed-application/fluxed-application/flux/flux.min.js

**Comment by ju5t** [comment]
You can probably solve that by adding a `<base>` tag to your `<head>`. Something along the lines of:

```html
<base href="https://my-domain.com/fluxed-application/" />
```

This can break your site so test things first. A different solution would be to prefix the APP_URL to the CSS and JS files, but that may have side effects for multi tenant applications.
